packages/react-app/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn run lint-staged
```

Comment:
Since we created #38 can we omit this file? Might be better to have related code with issues for completeness

Reply:
In my latest commit, I moved this to the root of the repo and updated it to run the right `yarn` command